packages/playground/website/src/components/modal/index.tsx

```diff
@@ -1,61 +1,28 @@
-import ReactModal from 'react-modal';
+import React from 'react';
+import { Modal } from '@wordpress/components';
 import css from './style.module.css';
+import { ModalProps } from '@wordpress/components/build-types/modal/types';
+import classNames from 'classnames';
 
-ReactModal.setAppElement('#root');
-
-interface ModalProps extends ReactModal.Props {
-	styles?: ReactModal.Styles;
+interface ModalComponentProps extends ModalProps {
+	small?: boolean;
 }
-export const defaultStyles: ReactModal.Styles = {
-	content: {
-		top: '50%',
-		left: '50%',
-		right: 'auto',
-		bottom: 'auto',
-		marginRight: '-50%',
-		transform: 'translate(-50%, -50%)',
-		width: 400,
-		maxWidth: '100vw',
-		zIndex: 200,
-		textAlign: 'center',
-		color: '#000',
-		border: '#000 1px solid',
-		borderRadius: '6px',
-		background: '#fff',
-	},
-	overlay: {
-		background: '#1e2327d0',
-		zIndex: 10,
-	},
-};
-export default function Modal(props: ModalProps) {
-	const styles = {
-		overlay: { ...defaultStyles.overlay, ...props.styles?.overlay },
-		content: { ...defaultStyles.content, ...props.styles?.content },
-	};
+export function ModalComponent({
```

Comment:
@bgrgicak, this was a follow-up to my review comment [here](https://github.com/ajotka/wordpress-playground/blob/4a74fe8bdd3f657ab686f869dc0d7b8ed4f47731/packages/playground/website/public/wordpress.html#L159):
> If we are still using these styles everywhere, maybe we should restore the components/modal/index module, apply the styles in that one place, and then go back to importing Modal from components/modal rather than directly from @wordpress/components.